### PR TITLE
scripts/check-unused-variables.sh: add

### LINKED ISSUE #218: `uefi-password.robot`  AttributeError: 'list' object has no attribute 'encode'
### Device

VP4650

### RTE version

Dasharo v1.2.0

### Affected component(s) or functionality

_No response_

### Brief summary

PSW002.001 Password setting mechanism correctness checking :: This... | FAIL | AttributeError: 'list' object has no attribute 'encode'

### How reproducible

_No response_

### How to reproduce

robot -L TRACE -v rte_ip:192.168.10.203 -v config:protectli-vp4650 -v device_ip:192.168.10.21 dasharo-security/uefi-password.robot 

### Expected behavior

PASS

### Actual behavior

AttributeError: 'list' object has no attribute 'encode'

### Link to screenshots or logs

[log-uefi-password.zip](https://github.com/Dasharo/open-source-firmware-validation/files/14538655/log-uefi-password.zip)

### Additional context

_No response_

### Solutions you've tried

_No response_

### PR DESCRIPTION
**Pull request by macpijan**
https://github.com/Dasharo/open-source-firmware-validation/issues/209

### CODE REVIEW

File: platform-configs/msi-pro-z790-p.robot [hunk 1]
```diff
@@ -1,347 +1,10 @@
 *** Settings ***
-Resource    ../os-config/ubuntu-credentials.robot
+Resource    include/msi-z690-common.robot
```
Comment (mkopec) [inline platform-configs/msi-pro-z790-p.robot:2]: @macpijan did you push that `include` directory? I can't find it here

Comment (macpijan) [reply platform-configs/msi-pro-z790-p.robot:2]: @mkopec let me see :smile:

Comment (macpijan) [reply platform-configs/msi-pro-z790-p.robot:2]: I think I skipped something as I have changed the name of this dir from `variables` to `include`. Commit might need some cleanup, but the idea should be already there

Comment (mkopec) [reply platform-configs/msi-pro-z790-p.robot:2]: Thanks, I'll try to follow with this idea for pcengines boards

**Comment by macpijan** [comment]
Still not reworked:
- [ ] NovaCustom
- [ ] qemu

**Comment by macpijan** [comment]
Integrated into develop as part of: https://github.com/Dasharo/open-source-firmware-validation/pull/219

**Pull request closed without merging**